config/src/comments.rs

```diff
@@ -249,11 +249,40 @@ fn comments() -> HashMap<String, String> {
 		.to_string(),
 	);
 
+	retval.insert(
+		"i2p_mode".to_string(),
+		"
+#Configuration for I2P support (\"Disabled\" by default)
+#I2P can be enabled by the following:
+#[server.p2p_config.i2p_mode]
+#mode = \"Enabled\"
+
+#[server.p2p_config.i2p_mode.i2p_config]
+#autostart = true/false  - start I2P automatically
```

Comment:
More admin-friendly config:
```toml
# Start I2P automatically
# autostart = true
# Exclusively connect through I2P, or also use TCP/IP
# exclusive = true
```

Reply:
I missed this before the most recent push of updates. Can add with next round of changes.